apps/privacy-panel/templates/tc/applications.html

```diff
@@ -8,7 +8,15 @@ <h1 data-l10n-id="tc-applications">Applications</h1>
     </header>
 
     <div>
-      <ul>
+      <form id="tc-appSearch" role="search">
+        <button type="submit" data-l10n-id="tc-search-close">close</button>
+        <p>
+          <input type="text" data-l10n-id="tc-search-apps" placeholder="Search Apps" dir="auto">
+          <button type="reset" data-l10n-id="tc-search-clear">Clear</button>
```

Comment:
Ugh, you’re right.

I think it should be `<input type="submit">` rather than `<button type="submit">`, but that’s how it woks in the building blocks. I’ll just add some JS for that.